### PR TITLE
fix venv, pinning dependencies

### PR DESCRIPTION
**Pull request by Rohitrajak1807**
@deepakgupta191199 please use
```python
python -m venv ./venv
```
to create a venv. The current setup installs packages under ~/.local/lib which may not be desired.
Pinning all package versions.

### CODE REVIEW

**Comment by deepakg202** [comment]
@Rohitrajak1807 Thanks for pointing that out. Can you also update the instructions in the readme ?

**Comment by Rohitrajak1807** [comment]
Okay

**Comment by Rohitrajak1807** [comment]
@deepakgupta191199 it's updated

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # Color Recognizer App
 Simple GUI app to detect colors from camera or an image file.
 
+## Requirements
+- [qt5-base](https://archlinux.org/packages/extra/x86_64/qt5-base/)
+
```
Comment (deepakg202) [inline README.md]: I don't think this is required as the pyqt5 wheel will install the qt libraries

**Comment by Rohitrajak1807** [comment]
It's required by py qt.

File: README.md (hunk 1 above)
Comment (Rohitrajak1807) [reply README.md]: Yes, rightly mentioned.

**Pull request merged**